radio_beam/beam.py

```diff
@@ -149,6 +149,33 @@ def from_aips_header(cls, hdr):
         else:
             return None
 
+    def attach_to_header(self, header, copy=True):
+        '''
+        Attach the beam information to the provided header.
+
+        Parameters
+        ----------
+        header : astropy.io.fits.header.Header
+            Header to add/update beam info.
+        copy : bool, optional
+            Returns a copy of the inputted header with the beam information.
+
+        Returns
+        -------
+        copy_header : astropy.io.fits.header.Header
+            Copy of the input header with the updated beam info when
+            `copy=True`.
+        '''
+
+        if copy:
+            copy_header = header.copy()
+
+            copy_header.update(self.to_header_keywords())
+
+            return copy_header
+
+        header.update(self.to_header_keywords())
```

Comment:
for consistency's sake, I think it's nice to have the function always return something